frontend/src/libs/components/modal/steps/delete-account/delete-account-message.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button } from '#libs/components/components.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  onNext?: () => void;
+  onClose?: () => void;
+};
+
+const DeleteAccountMessage: React.FC<Properties> = ({ onNext, onClose }) => {
+  const hasNextStep = onNext && typeof onNext === 'function';
+  const hasCloseButton = onClose && typeof onClose === 'function';
```

Comment:
I have tried to do that previously but always gives me this error:

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/107894952/3fb31440-cfd7-4e90-9c44-6f0edf9bd616)

I can solve it by adding the option of undefined to the onClick property on the button:

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/107894952/8e09bebd-4098-4b56-9276-2ab06773982e)

but I am not sure if adding undefined to the onClick will affect other functionality since it is a component that is used frequently. I will make the change but I had chosen before to specify the type to avoid changing the original button component and avoid any possible bug.